test/test_jit_cuda_fuser.py

```diff
@@ -168,6 +168,39 @@ def t(x: torch.Tensor, y: torch.Tensor, z: float):
         self.assertEqual(o, jit_o)
         self.assertGraphContains(t_jit3.graph_for(x, y, 2.0), FUSION_GROUP)
 
+    # Testing partition logic that is capable to avoid creating unsupported
+    # broadcasting semantics in CudaFusionGroup
```

Comment:
Seems this test is figuring out what it can generate, not that it avoids unsupported cases.

Reply:
That is true.
If the partition does create unsupported case, it would error out. So it sorta checks the right thing.
This is not ideal tho, I can put some more lines there to traverse the fused graph.

Reply:
Just FYI, getting the bailout subgraph is trickier than what I thought it would be. I'm going to split the test into two instead of using the BailOut thing :facepalm:

Reply:
I'm going to leave it as-is for now.
Realized that relying on bailOut test is a poor strategy. I'll do a follow up PR to overhaul some related tests as well.